pydantic/wrapper.py

```diff
@@ -0,0 +1,409 @@
+"""
```

Comment:
humm, is this really necessary, can't we implement a few methods like `__repr__`, `__eq__` etc. and get the all the functionality required?

This seems like it might confuse people (e.g. in trackbacks) and damage performance.

Reply:
if we do need this, please could you add an `__all__`?